src/darwin/frida-helper-backend-glue.m

```diff
@@ -2028,7 +2028,11 @@ static void frida_darwin_helper_backend_launch_using_lsaw (NSString * identifier
     {
       instance->info_ptr_address = gum_darwin_module_resolve_symbol_address (dyld, "__ZN5dyld412gProcessInfoE");
       if (instance->info_ptr_address == 0)
-        goto dyld_probe_failed;
+      {
+        instance->info_ptr_address = gum_darwin_module_resolve_symbol_address (dyld, "__ZL12sProcessInfo");
```

Comment:
Let's move this check first, for consistency with the existing logic where we check from newest to oldest.